caluma/caluma_core/tests/test_health_checks.py

```diff
@@ -0,0 +1,542 @@
+        management.call_command("migrate")
+
+    # get /healthz/ response and compare to previous snapshot
+    response = client.get("/graphql/healthz")
+    snapshot.assert_match(response.json())
+
+    # assert database check response
+    if working_db:
+        assert response.json()["databases"] == [{"default": {"ok": True}}]
+        assert response.json()["db models"] == [{"default": {"ok": True}}]
+        assert response.json()["db migrations"] == [{"default": {"ok": True}}]
+        assert response.status_code == 200
+    else:
+        assert response.json()["databases"] == [{"default": {"ok": False}}]
+        assert response.json()["db models"] == [{"default": {"ok": False}}]
+        assert response.json()["db migrations"] == [{"default": {"ok": False}}]
+        assert response.status_code == watchman_settings.WATCHMAN_ERROR_CODE
+
+
+def switch_user(user, pwd):
+    """
+    Switch database user to either 'tester' or 'caluma' and
+    re-connect to database
+    """
+
+    # check if valid user
+    users = {"tester", "caluma"}
+    if not user in users:  # pragma: no cover
+        raise Exception("Could not switch user: Invalid user name " + user)
+
+    # connect new user to database
+    connections["default"].close()
+
+    settings.DATABASES["default"]["USER"] = user
+    settings.DATABASES["default"]["PASSWORD"] = pwd
+
+    connections["default"].connect()
+
+
+def switch_user_tester():
+    """
+    Switch database user to 'tester' with password 'test'
+    """
+
+    switch_user("tester", "test")
+
+
+def switch_user_caluma():
+    """
+    Switch database user to 'tester' with password 'test'
+    """
+
+    switch_user("caluma", "caluma")
+
+
+def create_test_user():
+    """
+    Create new user 'tester' with privileges necessary for health checks
+    """
+
+    # get db connection
+    cursor = connections["default"].cursor()
+
+    # create user and grant privileges
+    cursor.execute("CREATE USER tester WITH PASSWORD 'test';")
+    cursor.execute("GRANT ALL ON caluma_logging_accesslog TO tester;")
+    cursor.execute(
+        "GRANT USAGE, SELECT ON SEQUENCE caluma_logging_accesslog_id_seq TO tester;"
+    )
+    cursor.execute("GRANT ALL ON django_migrations TO tester;")
+
+
+def remove_test_user():
+    """
+    Remove user 'tester' and revoke privileges for health checks
+    """
+
+    # get db connection
+    cursor = connections["default"].cursor()
+
+    # revoke privileges and remove user
+    cursor.execute("REVOKE ALL ON caluma_logging_accesslog FROM tester;")
+    cursor.execute(
+        "REVOKE USAGE, SELECT ON SEQUENCE caluma_logging_accesslog_id_seq FROM tester;"
+    )
+    cursor.execute("REVOKE ALL ON django_migrations FROM tester;")
+    cursor.execute("DROP USER tester;")
+
+
+def modify_privileges(sql):
+    """
+    Execute SQL statements to modify privileges
+    """
+
+    # retrieve db connection and execute sql statements
+    cursor = connections["default"].cursor()
```

Comment:
You can also access `django.db.connection.cursor()` directly (singular connection!). This is an alias for the default DB, and may be simple enough to avoid having this helper function (`modify_privileges()`) at all.